src/autogluon/bench/cloud/aws/batch_stack/lambdas/lambda_function.py

```diff
@@ -0,0 +1,186 @@
+import os
+import uuid
+from itertools import product
+
+import boto3
+import yaml
+
+aws_batch = boto3.client("batch")
+s3 = boto3.client("s3")
+
+
+def submit_batch_job(env: list, job_name: str, job_queue: str, job_definition: str):
+    container_overrides = {"environment": env}
+    response = aws_batch.submit_job(
+        jobName=job_name,
+        jobQueue=job_queue,
+        jobDefinition=job_definition,
+        containerOverrides=container_overrides,
+    )
+    print(response)
+
+
+def download_config(s3_path: str):
+    file_path = os.path.join("/tmp", s3_path.split("/")[-1])
+    bucket = s3_path.strip("s3://").split("/")[0]
+    s3_path = s3_path.split(bucket)[-1].lstrip("/")
+    s3.download_file(bucket, s3_path, file_path)
+    return file_path
+
+
+def upload_config(bucket: str, file: str):
+    file_name = f'{file.split("/")[-1].split(".")[0]}.yaml'
+    s3_path = f"configs/{file_name}"
+    s3.upload_file(file, bucket, s3_path)
+    return f"s3://{bucket}/{s3_path}"
+
+
+def save_configs(configs: dict, uid: str):
+    config_file_path = os.path.join("/tmp", f"cloud_configs_split_{uid}.yaml")
+    with open(config_file_path, "w+") as f:
+        yaml.dump(configs, f, default_flow_style=False)
+    return config_file_path
```

Comment:
Would recommend add docstrings. Could be follow-up PRs

Reply:
[Issue](https://github.com/autogluon/autogluon-bench/issues/3) added.